VolumeLogic/VolumeMixer.py

```diff
@@ -30,8 +52,7 @@ def set_application(self, index, application):
         :param application: Application whose volume is being controlled.
         """
         self.applications[index] = application
-        self.modified_index = index
-        self.notify_all(ChangedValue.APPLICATION)
+        self.notify_all((index, ChangedValue.APPLICATION))
```

Comment:
Changed the value being sent through update to be a tuple following the (Control, Action) pattern that will be implemented later on. I think we will need to discuss this moving forward as to whether the Action is necessary or not.